types/leaflet/leaflet-tests.ts

```diff
@@ -1,4 +1,4 @@
-import * as L from 'leaflet';
+import L = require('leaflet');
```

Comment:
I agree with @henrythasler -- leave this as-is to show that it doesn't break existing uses.